serviceconnector/src/createServiceConnector/createLinker.ts

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { KnownClientType } from "@azure/arm-servicelinker";
+import { AzExtTreeItem, AzureWizard, AzureWizardExecuteStep, AzureWizardPromptStep, ExecuteActivityContext, IActionContext, createSubscriptionContext, nonNullValue } from "@microsoft/vscode-azext-utils";
+import { AzureSubscription } from "@microsoft/vscode-azureresources-api";
+import * as vscode from 'vscode';
+import { AuthenticationListStep } from "./AuthenticationListStep";
+import { ClientListStep } from "./ClientListStep";
+import { ICreateLinkerContext } from "./ICreateLinkerContext";
+import { LinkerCreateStep } from "./LinkerCreateStep";
+import { LinkerNameStep } from "./LinkerNameStep";
+import { TargetServiceListStep } from "./TargetServiceListStep";
+
+export interface LinkerItem {
+    subscription: AzureSubscription,
+    id: string,
+}
+
+export async function createLinker(context: IActionContext & ExecuteActivityContext, item: LinkerItem | AzExtTreeItem, preSteps: AzureWizardPromptStep<ICreateLinkerContext>[] = [], runtime?: KnownClientType[]): Promise<void> {
+    const subscription = item instanceof AzExtTreeItem ? item.subscription : createSubscriptionContext(item.subscription); // For v1.5 compatibility
```

Comment:
Should be using `isAzExtTreeItem()` here instead of `instanceof`. Classes are unique per instance of the utils package.